pfsimulator/parflow_lib/nl_function_eval.c

```diff
@@ -41,6 +41,8 @@ typedef struct {
   double SpinupDampP1;      // NBE
   double SpinupDampP2;      // NBE
   int tfgupwind;           //@RMM added for TFG formulation switch
+  int seepage_patch_one;  //RMM added two optional seepage face BCs
```

Comment:
Is this value and int or a double.   Seems to be read as a double then converted to int?

GetDoubleDefault is used to read and the type in the keys.rst document is double.